### ISSUE TITLE
remove lazy loading

### ISSUE THREAD

**Issue by rymsha** [enonic/lib-react4xp#469]
lazy loading does not improve production performance where first request waits for engine to be initialized anyway.
It also may harm startup time, because lazy loading uses all CPU power at application startup slowing down all other work.

**Comment by espen42** [comment]
The improvement is for non-react4xp development. The point of lazyloading was not to improve performance of react4xp first-render, but to allow developers faster work on other rendering since eager-loading blocks the entire app on startup for longer (sometimees much longer).

**Comment by espen42** [comment]
But eager loading ensures that all the rendering workers are initialized simultaneaouslty, instead of adding a warmup time for each time a new worker is used for the first time, and for each entry.

IMO, lazy is good for dev work, eager for prod. Although this may change drastically depending on new architecture, implementation or graal.

**Comment by rymsha** [comment]
> But eager loading ensures that all the rendering workers are initialized simultaneaouslty

This is one of the factors that kills startup performance because too many engines compete for CPU resources. In production it does not improve performance because there are a few requests happening in parallel waiting for pool to initialize

**Comment by espen42** [comment]
Sure, but you get that period reduced to a one-time occurrence. Which makes it possible what we did on friday on SSB: they set serving to cashed by varnish until the warmup is complete. After that we can know all the workers will render immediately. Doing this lazily will make it uncertain if some end users will experience (possibly _minutes_ of) idle waiting for the thread to warm up and render.

**Comment by rymsha** [comment]
there is a different way to warm up endpoint - just call them internally. 
Or via `main.js`

Besides eager loading does not load "runnable assets"

**Comment by espen42** [comment]
Yeah. My suggestion would be to keep the flag around, to switch on lazy for dev work (unless graal is REALLY efficient), and otherwise make lib-react4xp trigger it internally on app startup.

**Comment by ComLock** [comment]
Remove app.cfg, remove lazy load param sent to Java.

**Comment by ComLock** [comment]
Since we don't know which component are pure client-side, eager loading may load too much and produce errors.

For instance `Emotion` used by `MUI`.
